react/components/EXPERIMENTAL_Table/SimpleTable/index.tsx

```diff
@@ -3,14 +3,23 @@ import React, { FC, memo } from 'react'
 import Header from './Header'
 import Rows from './Rows'
 import { NAMESPACES } from '../constants'
+import useTableContext from '../hooks/useTableContext'
+import Loading from './Loading'
 
 const SimpleTable: FC = () => {
+  const { loading } = useTableContext()
   return (
     <div id={NAMESPACES.TABLE} className="order-1 mw-100">
       <div className="overflow-x-auto">
         <div className="dt w-100" style={{ borderSpacing: 0 }}>
```

Comment:
Are all those `div`s required?

Reply:
Not really. I was able to remove one. Nice catch